Gems/Terrain/Assets/Shaders/Terrain/ClipmapComputeHelpers.azsli

```diff
@@ -169,15 +180,16 @@ ClipmapLevel CalculateClosestClipmapLevel(
     // The top clipmap's render distance
     float2 minRenderSize = maxRenderSize / pow(clipmapScaleBase, (float)(clipmapStackSize - 1u));
 
-    float2 clampedDistance = clamp(minRenderSize, abs(distanceFromViewPosition) * RenderDistanceBias, maxRenderSize);
+    float2 absDistance = abs(distanceFromViewPosition);
+    float2 clampedDistance = clamp(absDistance * RenderDistanceBias, minRenderSize, maxRenderSize);
     float2 distanceRatio = maxRenderSize / clampedDistance;
     // Due to clamping, the max result of log is (clipmapStackSize - 1u), which falls into the first clipmap
     clipmapLevel.m_closestLevel = clipmapStackSize - 1u - uint(floor(log(min(distanceRatio.x, distanceRatio.y))/log(clipmapScaleBase)));
 
     float renderDistance = maxRenderDistance / pow(clipmapScaleBase, (float)(clipmapStackSize - 1u - clipmapLevel.m_closestLevel));
     float blendingStart = renderDistance * BlendingStartFactor;
     float blendingEnd = renderDistance / RenderDistanceBias;
-    float blendingValue = max(clampedDistance.x, clampedDistance.y);
+    float blendingValue = max(absDistance.x, absDistance.y);
```

Comment:
As far as I can tell this is separate distances in X and Y? I don't think this is right - you don't want to be rendering the detail texture further away just because you happen to be looking down a diagonal instead of an axis, it should be a single true distance value no matter the direction.

Reply:
Euclidean distance would result in circles instead of square rings. Square rings should use max of the coordinate distance, also know as Chebyshev distance.
Using circles makes more sense but it will make it harder to define margins and the area to update.

Here is a fix for the bottom layer, where it won't have blending due to clamping.

Reply:
I don't think this needs to affect what's updated - you can still update the whole square, but what you draw can be based on actual distances. Long term it'd be better to do this based on gradients similar to how mip-mapping works.